mixer/adapter/redisquota/BUILD

```diff
@@ -0,0 +1,42 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Please remove this file, we no longer build with Bazel

Reply:
Removed